packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -183,7 +183,7 @@ export const activate = async (extensionContext: vscode.ExtensionContext) => {
     new DebugConfigurationProvider()
   );
   const checkpointsView = vscode.window.registerTreeDataProvider(
-    'sfdx.view.checkpoint',
+    'sf.force.view.checkpoint',
```

Comment:
Why this name change regression to put `force` back?

Reply:
:weird: idk what happened here. Good catch!